pkg/base/linuxutils/fs/fs_helper.go

```diff
@@ -316,18 +294,18 @@ func (h *WrapFSImpl) Unmount(path string) error {
 	return err
 }
 
-// DeviceFs detect FS from the provided device using lsblk --output FSTYPE
+// GetFSType detect FS from the provided device using lsblk --output FSTYPE
 // Receives file path of the device as a string
 // Returns error if something went wrong
-func (h *WrapFSImpl) DeviceFs(device string) (string, error) {
+func (h *WrapFSImpl) GetFSType(device string) (string, error) {
 	var (
-		cmd    = fmt.Sprintf(DetectFSCmdTmpl, device)
+		cmd    = fmt.Sprintf(GetFSTypeCmdTmpl, device)
 		stdout string
 		err    error
 	)
 	if stdout, _, err = h.e.RunCmd(cmd,
 		command.UseMetrics(true),
-		command.CmdName(fmt.Sprintf(DetectFSCmdTmpl, ""))); err != nil {
+		command.CmdName(fmt.Sprintf(GetFSTypeCmdTmpl, ""))); err != nil {
 		return "", fmt.Errorf("failed to detect file system on %s: %v", device, err)
```

Comment:
It's better to use [`%w` for error](https://go.dev/blog/go1.13-errors#wrapping-errors-with-w) instead of `%v`.

Reply:
Thank you for the reference! Fixed